src/libcollections/slice.rs

```diff
@@ -79,10 +79,15 @@
 #![doc(primitive = "slice")]
 #![stable(feature = "rust1", since = "1.0.0")]
 
+#[cfg(not(test))]
```

Comment:
These attributes are unfortunately kinda sprinkled and make this tough to read, I think it's fine for this module to have `#![allow(unused_imports)]` with a comment explaining about all the `#[cfg(not(test))]` code